ovh/resource_cloud_project_database_mongodb_user.go

```diff
@@ -63,10 +64,17 @@ func resourceCloudProjectDatabaseMongodbUser() *schema.Resource {
 			},
 			"roles": {
 				Type:        schema.TypeSet,
-				Description: "Roles the user belongs to (without authentication database)",
+				Description: "Roles the user belongs to with the authentication database",
 				Optional:    true,
 				Elem: &schema.Schema{
 					Type: schema.TypeString,
+					ValidateFunc: func(v interface{}, k string) (ws []string, errors []error) {
```

Comment:
instead of this error, couldn't you manually add the `@admin` part to the role when it is not present ? This way we would avoid the breaking change.

Reply:
I think for this one we won't be able to dodge the breaking change.
We should not imply the @admin for the customer as it determine on which database the user have rights. For this reason we would prefer customer to be fully aware of the value it will be set to.